ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v2/fileinput/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    cq:icon="attach"
+    jcr:primaryType="cq:Component"
+    jcr:title="Adaptive Form File Attachment (v2)"
+    jcr:description="Add a button to upload one or more files as form attachment."
+    sling:resourceSuperType="core/fd/components/form/base/v1/base"
```

Comment:
This should extend v1, can you incorporate all comments, I still see many comments not incorporated